util/mock/context.go

```
@@ -46,6 +49,11 @@ type Context struct {
 	pcache      *kvcache.SimpleLRUCache
 }
 
+// Execute implements sqlexec.SQLExecutor Execute interface.
+func (c *Context) Execute(ctx context.Context, sql string) ([]ast.RecordSet, error) {
+	return nil, nil
```

It's better to return an error for the not supported method.